packages/ui/components/SPage/Home.vue

```
@@ -0,0 +1,20 @@
+<script setup lang="ts">
+  useSeoMeta({
+    title: 'Home'
+  });
```

ESLint

[error] Missing trailing comma.

(style/comma-dangle)